src/elm/string.coffee

```diff
@@ -64,6 +73,15 @@ module.exports.PositionOf = class PositionOf extends Expression
     string = @string.execute(ctx)
     if not (pattern? and string?) then null else string.indexOf(pattern)
 
+module.exports.Matches = class Matches extends Expression
+  constructor: (json) ->
+    super
+
+  exec: (ctx) ->
+    [string, pattern] = @execArgs ctx
```

Comment:
Totally not your fault, but I'm just noticing now that the ELM for `SplitOnMatches` passes the arguments as two named arguments (`stringToSplit` and `separatorPattern`), but `Matches` passes the string and pattern arguments as an array of unnamed `operands`.

That kind of inconsistency drives me nuts.

Reply:
Completely agree.